subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheTaskSerializationIntegrationTest.groovy

```diff
@@ -155,10 +155,12 @@ class ConfigurationCacheTaskSerializationIntegrationTest extends AbstractConfigu
     def "restores task abstract properties"() {
         buildFile << """
             interface Bean {
-                @Internal
+                @Input
+                @Optional
```

Comment:
❓ Do we still want to keep these changes? I think not, right?

Reply:
Right, now that we support all custom types we can drop this.